spellbook/models/uniswap/ethereum/uniswap_ethereum_schema.yml

```diff
@@ -101,7 +101,7 @@ models:
       project: uniswap_v3
       contributors: masquot, soispoke
     config:
-      tags: ['ethereum','uniswap_v3','trades']
+      tags: ['ethereum','uniswap_v3', 'trades', 'uniswap']
```

Comment:
This just makes me think of something I've found useful for tags, is separating 'uniswap' and 'v3' (but adding uniswap_v3 is fine too)